src/app/exams/exams-view.component.ts

```diff
@@ -115,7 +115,7 @@ export class ExamsViewComponent implements OnInit, OnDestroy {
   }
 
   goBack() {
-    this.router.navigate([ '../' ], { relativeTo: this.route });
```

Comment:
@coder8102 Please revert the route back to `'../'`.  The back button from the ExamsView will also sometimes go back to a course.